### DISCUSSION TITLE
Not print results in the console

### DISCUSSION THREAD

**Discussion by guidoferreyra** [fonttools/fontbakery#3728] [Q&A] [answered]
Hi this is more a question than an issue. Is there a way to not print the results of the checks in the terminal and just the progress bar?

**Comment by felipesanches** [comment, marked as answer]
I think that's currently not possible. Tell us more about your specific need. What's the usage scenario you have in mind?

I know that Font Bakery diverges from a pretty common behaviour of UNIX tools that is to be absolutely silent unless there's some error. We do the opposite, but I understand than some people might want the tool to only output results to log files and nothing on `stdout`. Is that what you're looking for?

**Comment by guidoferreyra** [comment]
Hi Felipe,

> I understand than some people might want the tool to only output results to log files and nothing on stdout.

Is exactly that. Font Bakery is executed as a part of larger set of tests and proofs and since the profile is read from the output file I would like it to be a bit quieter. Is somewhat cosmetic but I was wondering if maybe there was an option already for that.

Thanks for you answer!
